modules/web/i18n/ko/messages.ko.xlf

```diff
@@ -1554,7 +1554,7 @@
       </trans-unit>
       <trans-unit id="6119876613180878188" datatype="html">
         <source>Dropped Capabilities</source>
-        <target state="new">Dropped Capabilities</target>
+        <target state="new">제거된 기능(Dropped Capabilities)</target>
```

Comment:
Why did you keep english version?

Reply:
I thought that there is no exact specific word for "Capabilities" ,  it looks better that for save it for prevent confuses.

Reply:
Hi @jc01rho 
Thank you for the contribution ! :)

There are few Korean terms that provide original English term in k8s/dashboard in order to deliver exact meaning of the term. 

In case of "Capabilities", I know "Capabilities" is not exactly matched with "기능", but it is acceptable translation. Not to distort existing UI, I think we need to minimize long texts.

If my opinion is acceptable to you, please remove the English section. :)

Reply:
sure. I'd agree.